tools/travis-build-and-push-docker.sh

```diff
@@ -21,23 +19,30 @@ export BUILDS=`pwd`
 export MEMBER_TGZ=mongooseim.tar.gz
 
 DOCKERHUB_TAG=${TRAVIS_BRANCH}
+VERSION=`tools/generate_vsn.sh`
+GIT_REF=`git rev-parse --short HEAD`
 
 if [ ${TRAVIS_PULL_REQUEST} != 'false' ]; then
     DOCKERHUB_TAG="PR-${TRAVIS_PULL_REQUEST}"
 elif [ ${TRAVIS_BRANCH} == 'master' ]; then
     DOCKERHUB_TAG="latest";
 fi
 
+IMAGE_TAG=${DOCKERHUB_USER}/mongooseim:${DOCKERHUB_TAG}
+
 git clone https://github.com/esl/mongooseim-docker.git
 cd mongooseim-docker
+git checkout 843558f
 
 cp ../${MONGOOSE_TGZ} member
 
-docker build -f Dockerfile.member -t mongooseim .
-
-docker login -e=${DOCKERHUB_EMAIL} -u ${DOCKERHUB_USER} -p ${DOCKERHUB_PASS}
+docker build -f Dockerfile.member -t ${IMAGE_TAG} \
+             --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
+	     --build-arg VCS_REF=${GIT_REF} \
+	     --build-arg VERSION=${VERSION} \
+	     .
 
-docker tag mongooseim ${DOCKERHUB_USER}/mongooseim:${DOCKERHUB_TAG}
+docker login -u ${DOCKERHUB_USER} -p ${DOCKERHUB_PASS}
 
 docker push ${DOCKERHUB_USER}/mongooseim:${DOCKERHUB_TAG}
```

Comment:
I guess it's better to use `${IMAGE_TAG}` here, since it's already used in command that creates this image.